content/en/docs/architecture/ci-operator.md

```diff
@@ -362,6 +364,44 @@ releases:
         upper: "4.5.0-0"
 {{< / highlight >}}
 
+### Testing with a Cluster from a Cluster Pool
+The `cluster_claim` below claims an `ocp` cluster with `4.7` version on `aws`. The owner of the `aws` account with which the cluster is installed is `dpp`. If the claimed cluster is running in `1h`, the environment variable `${KUBECONFIG}` stores the path to the `admin`'s `kubeconfig` file that can be used to contacting the cluster. The `kubeadmin`'s password is stored in `${KUBEADMIN_PASSWORD_FILE}`.
```

Comment:
```suggestion
The `cluster_claim` below claims an OCP 4.7 cluster in AWS from a pool owned by `dpp`. If the cluster is successfuly claimed from the pool, `ci-operator` executes the specified multi-stage test and provides it the credentials to access the cluster via two environmental variables:

 -  `${KUBECONFIG}`: Path to `system:admin` credentials.
 -  `${KUBEADMIN_PASSWORD_FILE}`: Path to the kubeadmin password file.
```

Reply:
Note: https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables should also be restructured to take claimed clusters into account, but let's do it in a followup.

Reply:
https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables  is still valid.
"the ephemeral OpenShift cluster" can be a claimed cluster as well, no?

Reply:
It's not entirely valid, because some variables are not available in steps running on claimed clusters.

Reply:
```console
url=https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/18007/rehearse-18007-pull-ci-openshift-ci-ns-ttl-controller-master-e2e/1387458618059657216/artifacts/build-resources/pods.json

$ curl -s $url | jq -r '.items[]|select(.metadata.name=="e2e-claim").spec.containers[0].env[]|.name'
BUILD_ID
CI
JOB_NAME
JOB_SPEC
JOB_TYPE
OPENSHIFT_CI
PROW_JOB_ID
PULL_BASE_REF
PULL_BASE_SHA
PULL_NUMBER
PULL_PULL_SHA
PULL_REFS
REPO_NAME
REPO_OWNER
ENTRYPOINT_OPTIONS
ARTIFACT_DIR
NAMESPACE
JOB_NAME_SAFE
JOB_NAME_HASH
KUBECONFIG
KUBEADMIN_PASSWORD_FILE
SHARED_DIR
```

I think this is the only one: `IMAGE_FORMAT ` (cross check with https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables), right?

Reply:
Oh, I see what you mean - some of them are basically ruled out by the "When is it present".